openfisca_france/parameters/prestations_sociales/solidarite_insertion/minima_sociaux/rsa/rsa_cond/duree_min_titre_sejour/eee.yaml

```
@@ -0,0 +1,20 @@
+description: Durée minimum du titre de séjour (pour un ressortissant de l'EEE) permettant de bénéficier du Revenu de solidarité active (RSA)
+values:
+  2009-06-01:
+    value: 0.25
+metadata:
+  ux_name: Ressortissant de l'EEE
+  unit: year
+  reference:
+    2009-06-01:
+    - title: Loi 2008-1249 du 01/12/2008, art. 3 (modif art. L262-4 du CASF)
+      href: https://www.legifrance.gouv.fr/affichTexteArticle.do?idArticle=LEGIARTI000019861526&cidTexte=JORFTEXT000019860428
+    - title: Décret 2009-404 du 15/04/2009, art. 2 (modif art. R262-2 du CASF)
+      href: https://www.legifrance.gouv.fr/affichTexteArticle.do?idArticle=LEGIARTI000020522717&cidTexte=JORFTEXT000020521930
+  official_journal_date:
+    2009-06-01: 2008-12-03; 2009-04-16
+  notes:
+    2009-06-01: 3/12 (3 mois)
+  description_en: "Active solidarity income (RSA): General conditions"
+documentation: |
+  Lien métier:  https://www.service-public.fr/particuliers/vosdroits/F19778
```

Ajouter un retour à la ligne ?
```suggestion
  Lien métier:  https://www.service-public.fr/particuliers/vosdroits/F19778

```